pkg/domain/vulnerability.go

```diff
@@ -301,7 +301,7 @@ func CountSeverityOccurrencesGeneric[T SeverityProvider](vulnerabilities []T) to
 			counts.Low++
 		case enums.SeverityTypeNone.String():
 			counts.None++
-		default:
+		case enums.SeverityTypeUnknown.String():
 			counts.Unknown++
```

Comment:
`[Nit]` We should still handle the default case as a safeguard:


```suggestion
case enums.SeverityTypeNone.String():
    counts.None++
case enums.SeverityTypeUnknown.String():
    counts.Unknown++
default:
    counts.Unknown++
```

Reply:
Ok